x-pack/plugins/observability/public/pages/alert_details/components/page_title.tsx

```diff
@@ -34,20 +34,26 @@ export interface PageTitleProps {
   alert: TopAlert | null;
 }
 
+export function pageTitleContent(ruleCategory: string) {
+  return (
+    <FormattedMessage
+      id="xpack.observability.pages.alertDetails.pageTitle.title"
+      values={{
+        ruleCategory,
+      }}
+      defaultMessage="{ruleCategory} {ruleCategory, select, Anomaly {detected} Inventory {threshold breached} other {breached}}"
```

Comment:
@maryam-saeidi I'm trying to parse what will be displayed here.

`select` and `detected`, `threshold breached` and `breached` look like variables, but they're not being passed in via the `values` prop?

Reply:
This logic was added in another PR, and I added some [tests](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/alert_details/components/page_title.test.tsx) in that PR too, which can help with understanding how it works.

In general, this is related to the **select** format that you can find more information about it [here](https://formatjs.io/docs/core-concepts/icu-syntax/#select-format).